src/misc/setup.cpp

```diff
@@ -1160,6 +1186,16 @@ bool CommandLine::FindString(char const * const name,std::string & value,bool re
 	return true;
 }
 
+bool CommandLine::FindStringOption(const std::string &name, std::string &value, bool remove) {
```

Comment:
All the same comments above for `FindInt` apply here too.

Reply:
Done.